playwright-e2e/pages/base-page.ts

```diff
@@ -54,7 +54,7 @@ export class BasePage {
   }
 
   async clickContinue() {
-    await this.continueButton.click();
+    await this.continueButton.click({clickCount:2, delay:300});
```

Comment:
Does that mean Playwright auto-waiting is not enough in this situation?
https://playwright.dev/docs/actionability

Wondering would it be better if we check everything in network is completed.
`await page.waitForLoadState('networkidle');`